trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './base_decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable_object.correct_name[0..9]
+  end
```

Comment:
Optional -
I strongly suggest you should create a function that checks if the character has a length greater than 10 it should trim the word to 9 👍🏾 💯